### PR TITLE
remove license docs about log_blooms.rs

### PR DESCRIPTION
**Pull request by ouwenkg**

### CODE REVIEW

File: cita-executor/core/src/libexecutor/executor.rs [hunk 1]
```diff
@@ -388,22 +386,6 @@ impl Executor {
...
-            .get(Some(DataCategory::Extra), &*position.get_index())
-            .map(|bytes| decode::<LogBloomGroup>(bytes.unwrap().as_slice()))
-            .expect("Get bloom group from db error.");
-
-        let bloom_group = log_bloom_group.into();
-
-        Some(bloom_group)
-    }
-}
-
```
Comment (ouwenkg) [inline cita-executor/core/src/libexecutor/executor.rs]: Another implementation already existed in [cita-chain](https://github.com/cryptape/cita/blob/develop/cita-chain/core/src/libchain/chain.rs#L257). For reviewers.

Comment (jerry-yu) [reply cita-executor/core/src/libexecutor/executor.rs]: add ours licence ？

File: cita-chain/types/src/header.rs [hunk 2]
```diff
@@ -34,7 +34,7 @@ use hashable::{Hashable, HASH_NULL_RLP};
 use util::HeapSizeOf;
 
 // from parity
-pub use crate::log_blooms::LogBloomGroup;
+// pub use crate::log_blooms::LogBloomGroup;
```
Comment (leeyr338) [inline cita-chain/types/src/header.rs]: Just delete this line?

File: cita-executor/core/src/libexecutor/executor.rs (hunk 1 above)
Comment (ouwenkg) [reply cita-executor/core/src/libexecutor/executor.rs]: I think there is no need to add such supplement docs.

In my respective, all the part of license docs in files should be deleted after we finish removing gpl. No matter it is `parity` or `cryptape`

File: cita-chain/types/src/header.rs (hunk 2 above)
Comment (ouwenkg) [reply cita-chain/types/src/header.rs]: I update again.

**Pull request merged**